src/ralph/conf.py

```diff
@@ -126,6 +132,7 @@ class MongoDatabaseBackendSettings(InstantiableSettingsItem):
 class DatabaseBackendSettings(BaseModel):
     """Pydantic model for database backend configuration settings."""
 
+    ASYNC_ES: AsyncESDatabaseBackendSettings = AsyncESDatabaseBackendSettings()
```

Comment:
That was my very first thought, but as async backends are very similar to their sync equivalent, and as we are even using the same docker image underneath, shouldn't we have the same settings for both async and sync version of a backend? And keep them under the same `database` directory? So keep async backends inside the DatabaseBackendSettings?
It will probably change with backend unification though.

Reply:
Sure, let's keep it this way)
That's a good point, if we keep the two-class approach for sync / async backends in the plugin architecture for backends we will indeed probably have separate settings.